platform/parsers/elastic_query_dsl/lucene/value.go

```diff
@@ -51,7 +51,12 @@ func (v termValue) toExpression(fieldName string) model.Expr {
 		if len(termAsStringToClickhouse) > 0 && (termAsStringToClickhouse[0] != '%' || termAsStringToClickhouse[len(termAsStringToClickhouse)-1] != '%') {
 			termAsStringToClickhouse = fmt.Sprintf("%%%s%%", termAsStringToClickhouse)
 		}
-		return model.NewInfixExpr(model.NewColumnRef(fieldName), "ILIKE", model.NewLiteralWithEscapeType(termAsStringToClickhouse, model.FullyEscaped))
+		if len(termAsStringToClickhouse) > 0 && (termAsStringToClickhouse[0] != '\'' || termAsStringToClickhouse[len(termAsStringToClickhouse)-1] != '\'') {
```

Comment:
It's not 100% same, but isn't here duplication of logic?
```suggestion
		if !utils.IsSingleQuoted(termAsStringToClickhouse) {
```